### PR TITLE
[IMP] mail: activity visual update

### PR DESCRIPTION
**Pull request by zel-odoo**
1. always show the name of the assignee.
2. replace the icon with avatar
3. bold the time info.

task-3142628

before:
image

after:
image

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by zel-odoo** [comment]
@alexkuhn I don't like the way we handle avatarUrl. In this case, if we want to get the proper avatar for the activity, we have to 
1. Use at least two extra services `persona` and `thread`, which are not needed for activity since we handle everything by the direct data from the model.
2.  The data provided from the server is user rather than partner/guest, it requires a new method that can get `Persona` with a user based on the `user_id`, which is complex.
3. Sometimes, the persona is even not fetched and created, but the data provided from server is just an id and there is no way to get the avatar rather than ask another fetch for the extra user data.

I think the most important thing is, sometimes we just want a simple avatarUrl and don't want to handle the `thread` and `persona` and maybe don't even want to create them. Maybe we should make avatar a service and direct provide separated functions for getting URL rather than putting everything together into `threadService`?

File: addons/mail/static/src/scss/variables/primary_variables.scss [hunk 1]
```diff
@@ -18,3 +18,5 @@ $o-mail-composer-text-input-height: 40px !default;
 $o-mail-LinkPreview-width: 320px !default;
 $o-mail-LinkPreview-height: 240px !default;
 $o-mail-LinkPreviewCard-height: 80px !default;
+
+$o-mail-ActivityListPopoverItem-assignee-avatar-size: 1.5rem !default;
```
Comment (alexkuhn) [inline addons/mail/static/src/scss/variables/primary_variables.scss:22]: No need for this variable, it's a very specific size, so can just put 1.5rem immediately where needed

**Comment by alexkuhn** [comment]
> @alexkuhn I don't like the way we handle avatarUrl. In this case, if we want to get the proper avatar for the activity, we have to
> 
> 1. Use at least two extra services `persona` and `thread`, which are not needed for activity since we handle everything by the direct data from the server and activity model.
> 2. The data provided from the server is user rather than partner/guest, it requires a new method that can get `Persona` with a user based on the `user_id`, which is complex.
> 3. Sometimes, the persona is even not fetched and created, but the data provided from server is just an id and there is no way to get the avatar rather than ask another fetch for the extra user data.
> 
> I think the most important thing is, sometimes we just want a simple avatarUrl and don't want to handle the `thread` and `persona` and maybe don't even want to create them. Maybe we should make avatar a service and direct provide separated functions for getting URL rather than putting everything together into `threadService`?

Good idea to improve code of `avatarUrl` by introducing an `avatar` service

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**